exi/src/lib.rs

```diff
@@ -84,8 +84,11 @@ impl SlippiEXIDevice {
             initial_dolphin_music_volume,
         } = config
         {
+            // TODO: Consider passing in user path directly and appending
+            let jukebox_path = self.config.paths.user_json.replace("user.json", "Jukebox");
```

Comment:
There might be a better way to do this. Thoughts @ryanmcgrath?

Reply:
This'll be easier once #18 is merged, since that migrates these kinds of things to true path types. Dunno if it's worth mucking with it here but I'll defer to whatever Nikki wants.